enterprise_subsidy/apps/core/management/commands/produce_enterprise_ping_event.py

```diff
@@ -0,0 +1,96 @@
+"""
+Produce a single event for enterprise-specific testing or health checks.
+
+Implements required ``APP.management.commands.*.Command`` structure.
+"""
+import logging
+import uuid
+
+import attr
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from edx_event_bus_kafka.internal.producer import create_producer
+from openedx_events.data import EventsMetadata
+from openedx_events.tooling import OpenEdxPublicSignal
+
+logger = logging.getLogger(__name__)
+
+
+# First define the topic that our consumer will subscribe to.
+ENTERPRISE_CORE_TOPIC = getattr(settings, 'EVENT_BUS_ENTERPRISE_CORE_TOPIC', 'enterprise-core')
+
+
+# Define the shape/schema of the data that our consumer will process.
+# It should be identical to the schema used to *produce* the event.
+@attr.s(frozen=True)
+class PingData:
+    """
+    Attributes of a ping record.
+    """
+    ping_uuid = attr.ib(type=str)
+    ping_message = attr.ib(type=str)
+
+
+ENTERPRISE_PING_DATA_SCHEMA = {
+    "ping": PingData,
+}
+
+# Define the key field used to serialize and de-serialize the event data.
+ENTERPRISE_PING_KEY_FIELD = 'ping.ping_uuid'
+
+# Define a Signal with the type (unique name) of the event to process,
+# and tell it about the expected schema of event data. The producer of our ping events
+# should emit an identical signal (same event_type and data schema).
+ENTERPRISE_PING_SIGNAL = OpenEdxPublicSignal(
+    event_type="org.openedx.enterprise.core.ping.v1",
+    data=ENTERPRISE_PING_DATA_SCHEMA
+)
```

Comment:
Are we going to define more than a small base set of signals in every repo, or is the idea they can be defined DRY in a shared repo?

Reply:
These signals are really for demo/testing purposes - our production-use signals will be defined in openedx-events: https://github.com/openedx/openedx-events/pull/347